README.md

```diff
@@ -1,13 +1,31 @@
-# opstools
+# Opstools
 
 Tools for Operations.
 
-# setup
+* __Point People:__ [#product-velocity][velocity_channel]
 
-Read/run the script:
+# Setup
 
-`./bin/setup`
+Read and run the setup script:
+```
+./bin/setup
+```
 
 Load environment vars:
+```
+export $(grep --no-filename --invert-match '^#' .env.shared .env | xargs)
+```
```

Comment:
Is that a `foreman` (from Ruby world) type of tool for Python?

Reply:
You could just use ruby's own `foreman` (like `foreman run ...detect.py`), but probably not worth the overhead of a whole ruby environment.

Reply:
oh! good to know.